packages/web/pages/user/my-account/bookmarks.js

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { useRouter } from 'next/router';
 import { useTranslation } from 'react-i18next';
+import { FiEye, FiX } from 'react-icons/fi';
 import { UserProfile } from '../../../components/UserProfile';
 import { Protected } from '../../../components/Authorization';
 import { getUserBookmarks } from '../../../services';
 import { Title } from '../../../components/Common';
 import { DataGrid } from '../../../components/DataGrid';
+import { IconButton } from '../../../components/Button';
 import { ORDERING as orderEnum } from '../../../utils/enums/api.enum';
+import { STATUS as dealStatusEnum } from '../../../utils/enums/technology.enums';
+
+export const getDealStatusText = (value) =>
```

Comment:
Shouldn't it be `technologyStatus` instead of `dealStatus`?

Reply:
Yep